releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,21 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+# spring-cloud-azure-appconfiguration-config-web
+
+[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
+
+## 1.1.2/1.2.2 - February 25, 2020
+
+* ```AppConfigurationRefresh```'s ```refreshConfigurations()``` can be called to check if the configurations are not up to date and if the cache has expired trigger a refresh event.
+* A new way of authenticating has been added. ```AppConfigurationCredentialProvider``` and ```KeyVaultCredentialProvider``` can be used to pass ```TokenCredentials``` to the provider to use when connecting with App Configuration and Key Vault. This method also allows for different authentication methods/credentials to be used when connecting with multiple App Configuration stores or Key Vaults.
+* The ```spring.cloud.azure.appconfiguration.stores[0].fail-fast``` setting has been updated to be per store. When fail-fat is set to false for an App Configuration instance, any failures thtat occur durring its initial load or subsequent refreshes will be ignored.
+  * Previously this setting on would either throw an error or attempt to deal with any error that occurred while trying to both start-up or refresh configurations.
```

Comment:
>Previously [](start = 4, length = 10)

"Previously this setting on". Do you mean "Previously when this setting was on"?